FNF_MissionTemplate.VR/client/loadout/fn_addItems.sqf

```diff
@@ -39,3 +39,25 @@ if (pRole in [ROLE_AAR,ROLE_AM,ROLE_CR]) then {player addWeapon phx_loadout_bino
 
 if (pRole == ROLE_RS) then {"ACE_SpottingScope" call phx_fnc_addGear};
 if (pRole == ROLE_SNP) then {"ACE_Tripod" call phx_fnc_addGear; "ACE_RangeCard" call phx_fnc_addGear;};
+
+if (
+    pRole in [
+        ROLE_PL,
+        ROLE_SL,
+        ROLE_P,
+        ROLE_CR
+    ]
```

Comment:
See here that the key privilege is based on player's class. Note that some non-leadership mission slots will be privileged (eg. platoon sergeant, all crewmen, all pilots). This is probably good enough to prevent unwanted vehicle claiming, but if we want to get more specific, we can.